README.md

```diff
@@ -79,6 +79,15 @@ from the folder where you cloned the ionosctl git.
 
 ### Authenticating with Ionos Cloud
 
+* Using environment variables
+If you have set the enviroment credentials, skip this section, if not, you can set them or follow the steps below for using config a config file:
+  
+You can set the enviroment variables using the next commands:
+```
+export IONOS_USERNAME="ionos username"
+export IONOS_PASSWORD="ionos password"
+```
+
```

Comment:
After today;s demo, I think it is important to add in here that environment variables have priority to config file. This applies also to the other README.md and to v6.

Reply:
changed